spring-cloud-starter-common-security-config-web/pom.xml

```diff
@@ -7,12 +7,36 @@
 		<version>1.8.0-SNAPSHOT</version>
 	</parent>
 	<artifactId>spring-cloud-starter-common-security-config-web</artifactId>
-
+	<properties>
+		<spring-security-oauth2.version>2.5.1.RELEASE</spring-security-oauth2.version>
+	</properties>
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-common-security-config-web</artifactId>
 			<version>1.8.0-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.security.oauth</groupId>
+			<artifactId>spring-security-oauth2</artifactId>
+			<version>${spring-security-oauth2.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.security</groupId>
+			<artifactId>spring-security-test</artifactId>
+			<version>5.6.2</version>
```

Comment:
Can we avoid putting the version here or Is there something required in `5.6.2` specifically?

Reply:
It wasn't required.